terraform/environments/digital-prison-reporting/modules/domains/replay-pipeline/pipeline.tf

```diff
@@ -83,9 +117,24 @@ module "replay_pipeline" {
               "--dpr.config.key" : var.domain
             }
           },
-          "Next" : "Move Archived Data Back to Raw Bucket"
+          "Next" : "Archive Remaining Raw Files"
         },
-        "Move Archived Data Back to Raw Bucket" : {
+        "Archive Remaining Raw Files" : {
```

Comment:
This step archives the last set of CDC raw files. This ensures the archive is complete.